src/commands/installDocker.ts

```diff
@@ -0,0 +1,56 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IActionContext } from 'vscode-azureextensionui';
+import { ext } from '../extensionVariables';
+import { localize } from '../localize';
+import LocalOSProvider from '../utils/LocalOSProvider';
+import { DockerInstallerBase } from './dockerInstallerBase';
+
+class WindowsDockerInstaller extends DockerInstallerBase {
+    protected downloadUrl: string = 'https://aka.ms/download-docker-windows-vscode';
+    protected fileExtension: string = 'exe';
+    protected installationMessage: string = localize('vscode-docker.commands.WindowsDockerInstaller.installationMessage', 'Installer is launched. Please follow the prompts to complete the installation, and then start the Docker Desktop.');
```

Comment:
```suggestion
    protected installationMessage: string = localize('vscode-docker.commands.WindowsDockerInstaller.installationMessage', 'Installer is launched. Please follow the prompts to complete the installation, and then start Docker Desktop.');
```
Sorry, I missed this in the first pass.